utils.js

```diff
@@ -74,6 +76,7 @@ function sendPing(key) {
     version: 1,
     tab_center_tabs_on_top: prefs.prefs.opentabstop,
     tab_center_show_thumbnails: prefs.prefs.largetabs,
+    tab_center_window_id: this.window.VerticalTabsWindowId,
```

Comment:
`this.window.VerticalTabsWindowId` successfully gets the id number.
I was planning on adding the current state see: todos above ;)